File: lösungen_lurch/PHP/wheeeee02.php
<?php

function duplicateHashtag($amount) {
	$hashtags = '';
	for ($i = 0; $i < $amount; $i++) {
		$hashtags .= "#";
	}

	return $hashtags;
}


// user input
$number = (int)readline("Enter MAX:");

$result = '';

for ($i = 1; $i <= $number; $i++) {
	$result = duplicateHashtag($i);
	print $result;
	print "\n";
}

for ($i = $number-1; $i >= 1; $i--) {
	$result = duplicateHashtag($i);
	print $result;
	print "\n";
}

File: lösungen_lurch/Python/wheeeee02.py
def duplicate_hashtag(amount):
    hashtags = ''

    for i in range(0, amount):
        hashtags += "#"

    return hashtags


# user input
number = int(input("Enter MAX:"))

for i in range(1, number+1):
    print(duplicate_hashtag(i))

for i in range(number-1, 0, -1):
    print(duplicate_hashtag(i))
